### ISSUE TITLE
Stash count sometimes returns 0 even though it's not

### ISSUE THREAD

**Issue by mortenfyhn** [mortenfyhn/bullet-train.zsh#2]

**Comment by mortenfyhn** [comment]
Might be only when stashes exist before adding the functionality to the prompt? In any case it happens because the stash count command used wrongly returns 0.

**Comment by mortenfyhn** [comment]
Might be that it only counts unnamed stashes?

**Comment by mortenfyhn** [comment]
This is it:

```
    # Use this to count stashes on current branch.
    # num_stashes=$(git stash list 2> /dev/null | grep "on `git rev-parse --abbrev-ref HEAD`" | wc -l)

    # Use this to count stashes in repo.
    num_stashes=$(git stash list 2> /dev/null | grep "stash@" | wc -l)
```

**Comment by mortenfyhn** [comment]
Fixed by 71bc59b.
